docs/product/user-feedback/index.mdx

```diff
@@ -86,3 +101,19 @@ If you don't have Sentry's default issue alert turned on, follow these steps to
 ![User Feedback Alert Configuration.](./img/user-feedback-alert-configuration.png)
 
 To get notifications when crash-report feedback comes in, make sure to turn on "Enable Crash Report Notifications" in Settings > Projects > [Project Name] > User Feedback.
+
+## Automated Issue Creation for User Feedback
+
+If you'd like to automatically create a GitHub or Jira issue every time you receive a user feedback, you can do so through the Alerts page in Sentry, similar to how you would create an issue alert for user feedback (illustrated in the section above).
```

Comment:
How about using the general terminology and using GitHub/Jira (the two major ones) as examples?

```suggestion
If you'd like to automatically create an external ticket, such as GitHub or Jira issues, every time you receive a user feedback, you can do so through the Alerts page in Sentry, similar to how you would create an issue alert for user feedback (illustrated in the section above).
```

Reply:
we only have options for github or jira i think, those are the only two options i saw in the alert settings

Reply:
ah nvm, those are just the only two on sentry!